### PR TITLE
Improvements to some classes, existence for others.

### PR DESCRIPTION
**Pull request by PseudoDistant**
I'm going to work on this more, it's not done yet.

### CODE REVIEW

**Comment by PseudoDistant** [comment]
OH MY GOD THE CHECKS!

**Comment by thecatcore** [comment]
Check failed because you created a conflict.

net/minecraft/class_794/getOrGenerateChunk
And
net/minecraft/class_476/loadChunk

Must have the same name, because they have the same intermediary name.

**Comment by thecatcore** [comment]
ChunkProvider and ClientChunkProvider

**Comment by PseudoDistant** [comment]
> Check failed because you created a conflict.
> 
> net/minecraft/class_794/getOrGenerateChunk And net/minecraft/class_476/loadChunk
> 
> Must have the same name, because they have the same intermediary name.

thonk

**Comment by PseudoDistant** [comment]
Screenshot_20211225_105423

**Comment by PseudoDistant** [comment]
Screenshot_20211225_105710

Cursed

**Review by thecatcore — APPROVED** [review]

**Comment by PseudoDistant** [comment]
I'm still not done with this, there's still more to map.

File: mappings/net/minecraft/world/biome/Biome.mapping [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+	FIELD field_12441 stoneBlockState Lnet/minecraft/class_2232;
+	FIELD field_12442 airBlockState Lnet/minecraft/class_2232;
+	FIELD field_12443 bedrockBlockState Lnet/minecraft/class_2232;
+	FIELD field_12444 gravelBlockState Lnet/minecraft/class_2232;
+	FIELD field_12445 redSandstoneBlockState Lnet/minecraft/class_2232;
+	FIELD field_12446 sandstoneBlockState Lnet/minecraft/class_2232;
+	FIELD field_12447 iceBlockState Lnet/minecraft/class_2232;
+	FIELD field_12448 waterBlockState Lnet/minecraft/class_2232;
+	FIELD field_12449 biomeList Lnet/minecraft/class_1942;
+	FIELD field_12450 biomeRegistry Lnet/minecraft/class_1943;
```
Comment (thecatcore) [inline mappings/net/minecraft/world/biome/Biome.mapping]:
```suggestion
	FIELD field_12450 REGISTRY Lnet/minecraft/class_1943;
```

**Review by thecatcore — CHANGES_REQUESTED** [review]

**Comment by PseudoDistant** [comment]
How does this all look?
Like should there be more work put into somewhere or can this be merged in?
(I can work more in a future PR, but I'd like to get a few of these changes in very soon.)

File: mappings/net/minecraft/world/biome/Biome.mapping (hunk 1 above)
Comment (PseudoDistant) [reply mappings/net/minecraft/world/biome/Biome.mapping]: Merged. :>

**Comment by PseudoDistant** [comment]
May as well rename this.

**Review by thecatcore — APPROVED** [review]

**Review by BluCobalt — APPROVED** [review]

**Comment by PseudoDistant** [comment]
@BoogieMonster1O1 AAAAAAAAAAAAAA

**Review by BoogieMonster1O1 — APPROVED** [review]

**Pull request merged**